### ISSUE TITLE
Chromium: installed app won't launch

### ISSUE THREAD

**Issue by tobiasBora** [NixOS/nixpkgs#150826]
### Describe the bug

I discovered today a really fun feature: installing web apps in chromium. For instance, if you browse to [https://xournal.online/](https://xournal.online/) on chromium, you can install the app locally:

![image](https://user-images.githubusercontent.com/2164118/146170216-3f1aeaad-e442-4adc-8864-2e2a0ffd947e.png)

Once installation is done, the website now runs as a separate application. You also should see on your desktop:

![image](https://user-images.githubusercontent.com/2164118/146170582-99454ceb-558b-4985-a7e5-2986a31a9a5f.png)

and in chrome://apps/ the app you installed. If you directly launch the app from chrome://apps/ it works directly. However, if you click on the app from the desktop, nothing happens. When inspecting the `.desktop file`, the reason is clear: it contains a line pointing to the unwrapped chromium:

```
Exec=/nix/store/76qscl06ahki1v240c9y76l4lixj45af-chromium-unwrapped-95.0.4638.54/libexec/chromium/chromium --profile-directory=Default --app-id=ncokflffjgpjncjjifkomecgbombmdgj
```

Instead, it should point to the wrapped chromium.

### Steps To Reproduce
1. Go to https://xournal.online/
2. Install the app (button in url bar)
3. Launch it from the shortcut added in your desktop (at least appears in KDE plasma)

### How to fix it

I'm not sure if there is a quick and dirty way to fix it. However, I think we should finally handle wrappers correctly in the whole Nix ecosystem. I'll create a new issue for that.

### Notify maintainers

@primeos @danielfullmer @Frostman @thefloweringash
